test/annexB/language/statements/try/catch-redeclared-for-of-var.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-variablestatements-in-catch-blocks
+es6id: B.3.5
```

Comment:
nic pick #2: this should be removed as this behavior is not compatible with ES6 anymore.